DatadogRUM/Sources/RUMMonitor/RUMCommand.swift

```diff
@@ -265,6 +265,17 @@ internal struct RUMAddCurrentViewMemoryWarningCommand: RUMErrorCommand {
     let missedEventType: SessionEndedMetric.MissedEventType? = .error
 }
 
+internal struct RUMAddViewLoadingTime: RUMCommand, RUMViewScopePropagatableAttributes {
+    var time: Date
+    var attributes: [AttributeKey: AttributeValue]
+    let canStartBackgroundView = false // no, it doesn't make sense to start "Background" view on receiving custom timing, as it will be `0ns` timing
+    let isUserInteraction = false // a custom view timing is not an interactive event
+
+    /// The name of the timing. It will be used as a JSON key, whereas the value will be the timing duration,
+    /// measured since the start of the View.
+    let missedEventType: SessionEndedMetric.MissedEventType? = nil
```

Comment:
question/ 🧠 ⚡ It might make sense to define another "event type" for missed `.viewLoadingTime`. This way we will aggregate count of loading times missed due to absence of active RUM view in addition to:

```swift
/// Represents types of event that can be missed due to absence of an active RUM view.
enum MissedEventType: String {
    case action
    case resource
    case error
    case longTask
}
```

Quality metrics for this are computed [here (internal)](https://dd-rum-telemetry.datadoghq.com/dashboard/u9k-m33-vde/mobie-metrics-rum-sessions?fromUser=true&index=*&refresh_mode=paused&tpl_var_source%5B0%5D=ios&view=spans&from_ts=1724676976266&to_ts=1724849776266&live=false&tile_focus=8402540209575996).

Reply:
that's reasonable, we can do this.